oar/core/worksheet.py

```diff
@@ -87,13 +87,13 @@ def create_test_report(self):
                 for k, v in candidate_builds.items():
                     build_cell_value += f"{k}: {v}\n"
             # if attr reference_releases! does not have anything, update the cell with empty
-            self._report.update_build_info(build_cell_value[:-1])
+            self._report.update_build_info(build_cell_value.strip())
             logger.info("build info is updated")
             logger.debug(f"build info:\n{build_cell_value}")
 
             # update shipment MRs
             shipment_cell_value = "\n".join(self._cs.get_shipment_mrs())
-            self._report.update_shipment_info(shipment_cell_value[:-1])
+            self._report.update_shipment_info(shipment_cell_value.strip())
```

Comment:
this is a bug, join func won't add '\n' or whitespace at the end of the string, suggestion is like below
```suggestion
            self._report.update_shipment_info(shipment_cell_value)
```

Reply:
`strip()` handles all those scenarios

Reply:
key point here is we don't need to handle this case for this line.

Reply:
In case the https://github.com/openshift/release-tests/blob/bd2add94efcfbee35509837a5f661734b93e7968/oar/core/configstore.py#L298 method returns correct data, we don't need to strip the `shipment_cell_value`

Reply:
Updated